app/src/main/kotlin/trees/gui/screens/AVLScreen.kt

```diff
@@ -0,0 +1,255 @@
+package trees.gui.screens
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import trees.BSTree
+import trees.dataBases.BST.insertAllNodesToTree
+import trees.dataBases.BST.removeFile
+import trees.dataBases.BST.writeAllNodesToFile
+import trees.nodes.BSNode
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.modifier.modifierLocalMapOf
+import main.kotlin.trees.dataBases.AVL.SQLiteDB
+import trees.AVLTree
+import trees.gui.*
+import trees.nodes.AVLNode
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun AVLScreen(toMenu: () -> Unit) {
+    val db = SQLiteDB("./app/src/main/kotlin/trees/dataBases/AVL/base.db")
+    val tree = remember { mutableStateOf(AVLTree<Int, String>()) }
+    var textMessage by remember { mutableStateOf(mutableListOf<String>()) }
```

Comment:
```suggestion
    val textMessage = remember { mutableStateListOf<String>()}
```
Из [документации](https://developer.android.com/jetpack/compose/state#use-other-types-of-state-in-jetpack-compose)